perma_web/frontend/components/CreateLink.vue

```diff
@@ -150,6 +157,15 @@ onBeforeUnmount(() => {
                     <LinkCount v-if="globalStore.userTypes.includes('individual')" />
                     <FolderSelect v-if="!globalStore.userTypes.includes('individual')" />
                 </fieldset>
+                <p v-if="!isToolsReminderSuppressed" id="browser-tools-message" class="u-pb-150"
+                    :class="globalStore.userTypes === 'individual' && 'limit-true'">
```

Comment:
The original legacy code added either `limit-true` or `limit-false` as classes to this paragraph depending on if a user was an individual. I noticed that there is no apparent `limit-false` class defined in the global css. So only `limit-true` is added here.